ros-tutorials/lesson-3-write-a-node.md

```diff
@@ -3,29 +3,35 @@
 Reminder: Read the notes!
 
 ## Summary
-In this lesson you will write your own node to control the turtle. This will involve making your own package,
-and writing a node inside that package. Moving forward, you will need to program in Python. If you do not have
-experience coding, you will need to learn Python. Find a tutorial/course you like online, or pick one from
-[here](https://stackify.com/learn-python-tutorials/#post-21937-_u49dnodg9ai6) or [here](https://gitconnected.com/learn/python).
-If you have experience coding in another language, feel free to proceed as Python should be fairly easy for you to pick up.
+In this lesson, you will write your own node to control the turtle. This will involve modifying files in a package we prepared for you in the starter code.
+
+This final tutorial will not necessarily provide all the information necessary. This is by design. The intention is to teach problem solving in the situation when you know _what_ you want to do, but not _how_ to do it. If there are things you do not recognize or feel comfortable with, take initiative and look them up online. Read tutorials, documentation, etc.
+
+:warning: Moving forward, you will need to program in Python. If you do not have experience coding, you will need to learn Python. Find a tutorial/course you like online, or pick one from [here](https://stackify.com/learn-python-tutorials/#post-21937-_u49dnodg9ai6) or [here](https://gitconnected.com/learn/python). If you have experience coding in another language, feel free to proceed, as Python should be fairly easy for you to pick up.
 
 ## Vocab
-- Package: A folder of nodes, messages, all related to some common goal or concept
-- `catkin`: The package managing system. This lets you create a workspace, create packages, manage packages, etc.
+- **Package**: A folder of nodes, messages, and other files all related to some common goal
 
 ## Goals
-1. Create your own package
-   - Be sure to use the newer versions of catkin commands (see Catkin Tools below). For example, do not use `catkin_make`, use `catkin build`. Similarly, `catkin_create_pkg` becomes `catkin create pkg`.
-2. Create a new node inside that package
-3. Write this node to make the turtle move in a square
-   - Hint: You will need to publish something somewhere
+1. Program a node inside that package by editing the premade Python file.
```

Comment:
Program the node to do what? I guess that's in the second step, but I kind of found that unclear.

Reply:
Good call, will clarify.